lua/acf/damage/damage_sv.lua

```diff
@@ -40,6 +40,43 @@ function Damage.getBulletDamage(Bullet, Trace)
 
 	return DmgResult, DmgInfo
 end
+
+--- Used to kill and fling the player because it's funny.
+--- Returns true if the damage has killed the player, false if it has not.
+function Damage.doSquishyFlingKill(Entity, Damage, HitPos, Attacker, Inflictor, Direction, Explosive)
+	local Health = Entity:Health()
+
+	if Damage > Health then
+		local SourceDamage = DamageInfo()
+		SourceDamage:SetAttacker(Attacker)
+		SourceDamage:SetInflictor(Inflictor)
+		SourceDamage:SetDamage(Damage)
+		SourceDamage:SetDamageForce(Direction * 200000000)
```

Comment:
I do like the body flinging, but a force multiplier of 200 million seems like a bit much?